IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetry.h

```diff
@@ -46,13 +47,19 @@ typedef NS_ENUM(NSInteger, TokenCacheRefreshType)
     TokenCacheRefreshTypeCachingMechanismNotImplemented,
 };
 
+#if !EXCLUDE_FROM_MSALCPP
+
 @interface MSIDCurrentRequestTelemetry : NSObject <MSIDTelemetryStringSerializable>
 
 @property (nonatomic) NSInteger schemaVersion;
 @property (nonatomic) NSInteger apiId;
-@property (nonatomic) TokenCacheRefreshType tokenCacheRefreshType;
 
+- (void)setTokenCacheRefreshType:(TokenCacheRefreshType)type;
```

Comment:
You can still have this as a property and call a setter method. Any reason for this change?

Reply:
Without this change, these pipeline tasks will fail: https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=947398&view=results

Reply:
Do you know why?

Reply:
@fengga, I'd still like to understand if this change is actually needed.

Reply:
I tested locally it is passing (I'm using xcode 13.3), but it will fail in the pipeline.
I tried a lot of ways to avoid it by changing the definition of the MACROs, 
#define CONDITIONAL_SET_REFRESH_TYPE(x, y) CONDITIONAL_COMPILE_MSAL_CPP([x setTokenCacheRefreshType:y])
#define CONDITIONAL_SET_REFRESH_TYPE(x, y) CONDITIONAL_COMPILE_MSAL_CPP([ x setTokenCacheRefreshType: y ])
#define CONDITIONAL_SET_REFRESH_TYPE(x, y) CONDITIONAL_COMPILE_MSAL_CPP([((x)) setTokenCacheRefreshType:((y))])
...
but none of them worked.
I think it's a bug in the version of xcode that the pipeline is using.

Reply:
It's passing now, thanks!